### ISSUE TITLE
Show only valid codes!

### ISSUE THREAD

**Issue by Aro-fox** [logicguy1/Discord-Nitro-Generator-and-Checker#77]
I would like an option someone in the text to either enable or disable invalid codes.
It gets annoying scrolling through all the codes to not even find a valid one. Even if there was It didn't pop up in the valid codes section.

**Comment by logicguy1** [comment]
When and if the script finds a valid code it will save it to a file and I believe that it should also stop checking codes

**Comment by abhishekhbhootwala** [comment]
> When and if the script finds a valid code it will save it to a file and I believe that it should also stop checking codes

I ran this script for 2h, no working code! Does it really work?

**Comment by vRobM** [comment]
Ran it for 99,999 codes, 0 work.
D must have changed their algo.

**Comment by Ryrin45** [comment]
Yeah about how often should we expect it to find a valid code? Is this something we should expect to take hours or maybe never even find one?

**Comment by Bennitenni111** [comment]
Its really hard to find valid codes with generator & checker.
instead spend some few bucks and buy it, or cheap nitros are available https://anonix.xyz/discord here, you can contact me.

**Comment by Mrx-dev** [comment]
if you want nitro code ,just buy one there's so small chance to get valid nitro code , the code just do random string its no way you can get valid nitro code :3
